Gemfile

```diff
@@ -2,4 +2,6 @@ source 'https://rubygems.org'
 
 gem 'sinatra'
 gem 'sequel'
-gem 'pg'
\ No newline at end of file
+gem 'pg'
+gem 'bcrypt'
+gem 'sinatra-flash'
```

Comment:
You can add rerun here too like:

```ruby
group :development do
  gem 'rerun'
end
```

Any gems that are used only in development should go under this group